core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -406,6 +413,11 @@ trait WhiskWebActionsApi
                         requestMethodParamsAndPath { context =>
                             provide(fullyQualifiedActionName(actionName)) { fullActionName =>
                                 onComplete(verifyWebAction(fullActionName, onBehalfOf.isDefined)) {
+                                    case Success((actionOwnerIdentity, action)) if context.method == OPTIONS &&
```

Comment:
i think i prefer to move the `if` predicate inside a single case statement for the `Success((actionOwnerIdentity, action))`; the predicate is sufficiently long I don't think we gain any readability or benefit from this.